test/async-hooks/init-hooks.js

```diff
@@ -212,13 +212,13 @@ class ActivityCollector {
 }
 
 exports = module.exports = function initHooks({
-    oninit,
-    onbefore,
-    onafter,
-    ondestroy,
-    allowNoInit,
-    logid,
-    logtype } = {}) {
+  oninit,
```

Comment:
I guess this is very C++, but not my cup of tea.
If there are new objections I'd rather have
```js
exports = module.exports = function initHooks(
  { oninit, onbefore, onafter, ondestroy, allowNoInit, logid, logtype } = {}) {
```